packages/core/package.json

```diff
@@ -23,9 +23,11 @@
     "prettier": "prettier --write src/**/*.ts"
   },
   "dependencies": {
+    "@ai-sdk/openai": "^0.0.40",
     "@latitude-data/compiler": "workspace:^",
     "@latitude-data/env": "workspace:^",
     "@t3-oss/env-core": "^0.10.1",
+    "ai": "^3.2.42",
```

Comment:
Does it need to go in core?

Reply:
yeah it will be used in wen and api

Reply:
If we use the sdk to generate stuff in web, then we only need `ai` in api.

Reply:
I actually don't think it's a good idea to use the sdk from web, since the sdk points to the production gateway and you don't want to have users do tests in production systems.

Reply:
But users in our web are also in production no?

Reply:
but you are doing tests with new prompts that might cause shenanigans, i prefer to sandbox local tests in the dashbaord rather than using the production gateway that affects all users